src/main/super/java/time/zone/TzdbZoneRulesProvider.java

```diff
@@ -243,7 +243,7 @@ ZoneRules createRule(short index) throws Exception {
 			ZoneRules rule = ruleData[index];
 			if (rule == null) {
 				ArrayBuffer arrayBuffer = encodeRuleData[index];
-				rule = (ZoneRules) Ser.read(ByteBuffer.wrapArrayBuffer(arrayBuffer));
+				rule = (ZoneRules) Ser.read(org.gwtproject.nio.TypedArrayHelper.wrap(arrayBuffer));
```

Comment:
The change is due to me using different gwt-nio, but since the TypeArrayHelper is there this is the only part that I might propose to be changed for next gwt-time release.

Reply:
Please, check the GWT29 branch if it will work for you. It is the only SHAPSHOT for now, but I am going to release missing projects `gwt-regexp`, `gwt-typedarrays` and `gwt-xhr` from https://github.com/gwtproject under `org.jresearch.gwtproject` group.